jkube-kit/enricher/specific/src/test/java/org/eclipse/jkube/kit/enricher/specific/AbstractHealthCheckEnricherTest.java

```diff
@@ -65,14 +70,15 @@ public void enrichSingleContainer() {
         list.accept(new TypedVisitor<ContainerBuilder>() {
             @Override
             public void visit(ContainerBuilder container) {
-                assertNotNull(container.build().getLivenessProbe());
-                assertNotNull(container.build().getReadinessProbe());
-                assertNotNull(container.build().getStartupProbe());
+                assertThat(container.build())
+                    .hasFieldOrProperty("livenessProbe")
+                    .hasFieldOrProperty("readinessProbe")
+                    .hasFieldOrProperty("startupProbe");
```

Comment:
Sorry, but I just saw this one.

This is **not** asserting that the value/probe is not null, it just checks that the field is there (which is always there)

If the values are static, then it would be feasable to check on one of the properties of the probe: `.hasFieldOrPropertyWithValue("livenessProbe.timeOutSeconds", 1337)`